### PR TITLE
Upgrade to Cassandra 3.0.10

### PR DESCRIPTION
**Pull request by cooldoger**

### CODE REVIEW

File: universe/resource.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "scheduler-zip":"{{artifact-dir}}/scheduler.zip",
       "executor-zip":"{{artifact-dir}}/executor.zip",
       "jre-tar-gz":"https://downloads.mesosphere.com/cassandra/assets/server-jre-8u74-linux-x64.tar.gz",
-      "apache-cassandra-bin-tar-gz":"https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.9-bin-dcos.tar.gz"
+      "apache-cassandra-bin-tar-gz":"https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.10-bin-dcos.tar.gz"
```
Comment (verma7) [inline universe/resource.json]: @mohitsoni Just a reminder that this needs to be built and uploaded by the Mesosphere team.

**Comment by mgummelt** [comment]
Is this urgent?  It looks like 3.0.10 hasn't made it's way to https://archive.apache.org/dist/cassandra/ yet, which is where we fetch distributions from.

**Comment by verma7** [comment]
This is not urgent.

It has made it's way here: http://www.apache.org/dyn/closer.lua/cassandra/3.0.10/apache-cassandra-3.0.10-bin.tar.gz from the official Cassandra downloads page (http://cassandra.apache.org/download/).

**Comment by mohitsoni** [comment]
@mesosphere-ci retest this please

**Comment by verma7** [comment]
Ping!

**Comment by triclambert** [comment]
@mesosphere-ci retest this please

**Comment by triclambert** [comment]
We're targeting another release this Wednesday, although it may get delayed by #328.

**Comment by verma7** [comment]
@mesosphere-ci retest this please

**Comment by triclambert** [comment]
@mesosphere-ci retest this please

**Comment by mohitsoni** [comment]
@mesosphere-ci retest this please

File: universe/resource.json (hunk 1 above)
Comment (mohitsoni) [inline universe/resource.json]: @verma7 We got apache-cassandra-3.0.10-bin-dcos.tar.gz uploaded to downloads.

**Comment by mohitsoni** [comment]
Thanks @cooldoger and @verma7 for the PR!

**Pull request merged**